css/cart.css

```diff
@@ -0,0 +1,65 @@
+@import '/css/nav.css';
+
+:root{
```

Comment:
we can create a default/common CSS file for root variable names & import it here. later on, it'll easier to change the global styles in just one file.

Reply:
Understood! Thanks for the review, I'll refactor the code once I'm done with the submission.